src/main/main.ts

```diff
@@ -3,6 +3,9 @@ import {initBLS} from "@chainsafe/bls";
 
 import {createWindow} from "./gui/window";
 import {DatabaseIpcHandler} from "./db/ipc";
+import {initSentry} from "./sentry";
+
+initSentry();
```

Comment:
I think we should leave user option on onboarding whether he would like to report those stuff or not. Since it could leak some privacy sensitive data

Reply:
Yep, could be GDPR-ish too. Will finally get use of that Consent window